paper-dialog.html

```diff
@@ -18,10 +18,11 @@
 `<paper-dialog>` is a dialog with Material Design styling and optional animations when it is
 opened or closed. It provides styles for a header, content area, and an action area for buttons.
 You can use the `<paper-dialog-scrollable>` element (in its own repository) if you need a scrolling
-content area. See `Polymer.PaperDialogBehavior` for specifics.
+content area. Auto focusing can be set to any child using the `autofocus` attribute.
```

Comment:
How about "To autofocus a specific child element after opening the dialog, give it the `autofocus` attribute"?